ganga/GangaCore/GPIDev/Lib/Tasks/ITransform.py

```diff
@@ -490,11 +646,6 @@ def __setattr__(self, attr, value):
 
             if value != []:
 
-                if getConfig('Output')['ForbidLegacyOutput']:
```

Comment:
And here

Reply:
Fair enough. I will fix it over the weekend.